static/build.yml

```diff
@@ -3,11 +3,10 @@ dependencies:
   - pip
   - pip:
     - ..  # d2l
-    - git+https://github.com/d2l-ai/d2l-book
```

Comment:
Let's keep this dependency since we may keep upgrading it to enable new features

Reply:
For some reason when I setup build.yml with d2l-book dependency, I get an error with sphinx not being available. It is collected by pip but mxtheme requires sphinx and hence errors out.

Reply:
Then we may need to fix it if this is an issue with the current source of d2l-book